deploy/docker/README.md

```diff
@@ -0,0 +1,18 @@
+######################
+
+### pump.io http://pump.io Docker image - listens port 443
+
+1) Copy pump.io.json.sample to pump.io.json and configure pump.io.json according to the needs
```

Comment:
I understand what you want... I'll redo the dockerfile with env variables - that is the only alternative option that I am aware of to enable something like that. For example ` --env PUMPIO_DBNAME="something"`.